app/web_conn.go

```diff
@@ -166,6 +168,7 @@ func (a *App) NewWebConn(cfg *WebConnConfig) *WebConn {
 	}
 
 	wc := &WebConn{
+		ID:                 model.NewId(),
```

Comment:
Arg :facepalm:, I saw the `wc.SetConnectionID(cfg.ConnectionID)` that we're doing while creating the `wc` and thought that the value was static. Will update :)

Reply:
Hmm .. wait, it is static across reconnects, but only for a single client session. i.e - every client session across websocket reconnects will have the same connectionID. Since you will be using the webapp's client code, it relates with the reliable websockets. A reliable websocket works across disconnects, so a virtual connection id refers to a reliable websocket.

As long as you are referring to a single websocket session from your code, using the `connectionID` should be fine.